ROADMAP.md

```diff
@@ -0,0 +1,33 @@
+# Kubeflow Distributed Training Operators 2020 Roadmap
+
+This document outlines the main directions on the Kubeflow Distributed Training Operators in 2020.
+
+## Maintenance and reliability
+
+We will continue developing capabilities for better reliability, scaling, and maintenance of production distributed training experiences provided by operators.
+
+* Enhance maintainability of operator common module https://github.com/kubeflow/common/issues/54
+* Migrate operators to use kubeflow/common apis
+* Graduate MPI Operator, Mxnet Operator and XGBoost Operator to v1
+
+## Features
+
+To take advatanges of other capabilities of job scheduler components, operators will expose more APIs for advanced scheduling. More features will be added to simplify usage like dynamic volume supports and git ops experiences. In order to make it easily used in the Kubeflow ecosystem, we can add more launcher KFP components for adoption.
+
+* Support dynamic volume provisioning for distributed training jobs https://github.com/kubeflow/common/issues/19
+* MLOPS - Allow user to submit jobs using Git repo without building container images.
+* Add Job priority and Queue in SchedulingPolicy for advanced scheduling in common operator https://github.com/kubeflow/common/issues/46
+* Add pipeline launcher components for different training jobs. https://github.com/kubeflow/pipelines/issues/3445
+
+
+## Monitoring
+
+* Provides a better common logger https://github.com/kubeflow/common/issues/60
+* Expose generic prometheus metrics in common operators https://github.com/kubeflow/common/issues/22
+* Centralized Job Dashboard for training jobs (Add metadata graph, model artifacts later)
```

Comment:
This seems to overlap with some existing projects?

Reply:
KF had a tf dashboard before but deprecated. Are you talking about that one?

Reply:
There was a discussion earlier regarding Job dashboard.  It was deprecated because users didn't find dashboard much useful.  And it is not possible to list all operator spec options in the UI.

Reply:
One thing can be, about a common dashboard across operators?

Reply:
Yeah. I think that would be helpful. We there's enough features for that common dashboard, user will like it. 

I think reasonable request will look like
1. Check jobs of all CRDs
2. basic metadata of the job -> start/end time. job name. resources. 
3. actions -> terminating job
4. metrics -> it would be great to integrate `kubeflow/metadata` info directly in that job, like model artifact, log params. Actually I think we can extend metadata SDK to more rich graphs like line chart, bar graph. It will be helpful if we have job level metrics